### DISCUSSION TITLE
add a way to connect a github website to a pterodactyl panel

### DISCUSSION THREAD

**Discussion by TitaniumTries** [pterodactyl/panel#3860] [Feature Requests]
like, everytime you make a change in the github repository, the server automatically updates itself

**Comment by DomiiBunn** [comment]
This is possible thrugh a [CI ](https://github.com/marketplace/actions/sftp-deploy)or an [Egg](https://github.com/parkervcp/eggs) (Some like DiscordJS support it nativley or you can creat a custom one)
